### PR TITLE
API-164  [Java SDK] ApiFactory: Loading bynder-sdk.properties fails in Tomcat

### PR DESCRIPTION
**Pull request by githuib**
Fixes https://github.com/Bynder/bynder-java-sdk/issues/38

### CODE REVIEW

File: src/main/java/com/bynder/sdk/util/Utils.java [hunk 1]
```diff
@@ -50,4 +52,20 @@ public static boolean isDateExpiring(final Date date, final int seconds) {
...
     }
+
+    /**
+     * Loads a (.properties) config file from the resources.
+     *
+     * @param file the filename of the config file (without the extension)
+     * @return {@link Properties} with the values from the config file
+     * @throws IOException if the config file could not be found
+     */
+    public static Properties loadConfig(final String file)
```
Comment (bankras) [inline src/main/java/com/bynder/sdk/util/Utils.java:63]: This deserves a unit test

File: src/main/java/com/bynder/sdk/sample/AppSample.java [hunk 2]
```diff
@@ -13,77 +13,121 @@
...
 import java.net.URL;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Properties;
 import java.util.Scanner;
 
 /**
  * Sample class to display some of the SDK functionality.
  */
 public class AppSample {
```
Comment (bankras) [inline src/main/java/com/bynder/sdk/sample/AppSample.java:39]: I'm not sure why this file is involved in this PR

File: src/main/java/com/bynder/sdk/util/Utils.java (hunk 1 above)
Comment (githuib) [reply src/main/java/com/bynder/sdk/util/Utils.java:63]: Thought of that, but what would we want to test exactly though? There's no real logic here, except stuff that's handled by other classes, like loading a .properties file into a Properties object.

File: src/main/java/com/bynder/sdk/sample/AppSample.java (hunk 2 above)
Comment (githuib) [reply src/main/java/com/bynder/sdk/sample/AppSample.java:39]: Could have only done the change of using the `Utils.loadConfig()` method. But I had this code already anyway. Could move it to a separate PR but personally didn't feel like this PR wasn't too complex anyway.

File: src/main/java/com/bynder/sdk/util/Utils.java (hunk 1 above)
Comment (bankras) [reply src/main/java/com/bynder/sdk/util/Utils.java:63]: You're offering a way to load a .properties file from a specific location. It makes sense to have that verified in a unit test. Could be as simple as a .properties file in `/src/test/resources/com/bynder/sdkutil/` that you try to read. Or maybe should it be `/src/test/resources/`? I always forget the exact location.

Comment (githuib) [reply src/main/java/com/bynder/sdk/util/Utils.java:64]: Added some unit tests for this method.

File: src/main/java/com/bynder/sdk/sample/AppSample.java (hunk 2 above)
Comment (githuib) [reply src/main/java/com/bynder/sdk/sample/AppSample.java]: Moved it out of this PR

**Review by bankras — APPROVED** [review]

**Pull request merged**